lucille-plugins/lucille-api/src/test/java/endpoints/ConfigInfoTest.java

```diff
@@ -0,0 +1,74 @@
+package endpoints;
+
+import static org.junit.Assert.*;
+
+import java.util.Optional;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.kmwllc.lucille.AuthHandler;
+import com.kmwllc.lucille.endpoints.ConfigInfo;
+
+import jakarta.ws.rs.core.Response;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConfigInfoTest {
```

Comment:
should add a test that actually looks at the json returned for a specific stage and confirms that the fields and field descriptions are as we would expect